Jenkinsfile

```diff
@@ -23,7 +23,7 @@ pipeline {
     }
     triggers {
         upstream(
-            upstreamProjects: 'SDK/ton-labs-abi/master, walletapi/sdk-emulator/feature-address-serialize',
+            upstreamProjects: 'Node/ton-labs-abi/master,walletapi/sdk-emulator/feature-address-serialize',

```

Comment:
1) The goal is to move whole dependency from other projects to `Node/ton-labs-abi/master`; do we really still need to keep  `walletapi/sdk-emulator/feature-address-serialize` in upstreams?
2) Does linker itself support this dependency change? Looks like not:
```#20 51.55    Compiling ton_abi v0.1.0 (https://github.com/tonlabs/TON-SDK.git#26a9474d)```
https://builder.tonlabs.io/jenkins/blue/organizations/jenkins/TVM-linker/detail/upstream_changes/3/pipeline#log-316

Reply:
1. I don't really know. I'm trying not to break anything that is working well now.
2. It seems like Nikita hasn't made changes in TVM-linker yet. ton-labs-abi was been created at the end of previous week, so I think Nikita will do it in the nearest time.